ci/cloudbuild/builds/integration-daily.sh

```diff
@@ -48,11 +48,11 @@ integration::bazel_with_emulators test "${args[@]}" "${integration_args[@]}"
 
 io::log_h2 "Running Bigtable integration tests (against prod)"
 bazel test "${args[@]}" "${integration_args[@]}" \
-  --test_tag_filters="integration-test" -- \
+  --test_tag_filters="integration-test,-ud-only" -- \
```

Comment:
Not all `-ud-only` added in this PR are necessary. This one is an example, we don't have tests for universe domain in `bigtable` directory. But by adding `-ud-only` with `integration-test` everywhere, it can remind us that we don't run domain universe tests with integration tests.

So I am open to keeping these "redundant" `-ud-only`s or remove them.

Reply:
I agree that it is best to add `-ud-only` everywhere.